ETC.tsv

```diff
@@ -17665,41 +17665,41 @@ ETC_20151224_017666	In substance, this is the kind of Golem and it is a creature
 ETC_20151224_017697	It gets created when the stone pieces or the stone figures of the buildings which the demons stayed change.
-ETC_20151224_017698	Bloks just appeared within the last 1 year so I haven't obtained much info yet.
-ETC_20151224_017699	There are some reckless people who are going after the monsters that have big ores like Canols attached onto the bodies of the monsters. It would have been better if they had acted more cautiously when thinking about the disaster that occured to the world.
-ETC_20151224_017700	He has four strong arms. If you don't be careful, you would receive lots of damages.
-ETC_20151224_017701	They are the demons that follow the demon lord, Hauberk. Most of them still show their loyalty to Hauberk even after their owners got ripped off and got sealed by Helgasercle.
-ETC_20151224_017702	Hohen competed against the humans from the birth of the kingdom. They were cooperative to demons instead of the goddesses, but they posed no big problem since their number was small compared to humans. However, the environment which Medizo Diena changed is working more advantegeous for Hohens instead of humans so it is not certain whether humans would have an edge over them in the future.
-ETC_20151224_017703	Unlike other specter type monsters, Leapers used to work for a long time in the demon world and it's evil whereabouts are recored in the kingdom's history.
-ETC_20151224_017704	The demon lord, Hauberk singed secret treaty with Agailla Flurry who is known as one of the greatest wizards in the history, but after that treaty had been discovered by the demon lord, Helgasercle, it's body was ripped in pieces and those pieces were sealed and scattered across the world. Nevertheless, Hauberk still influnces it's subordinates and dreams for the resurrection and it's return.
+ETC_20151224_017698	Bloks just appeared within the last year, so there isn't much information about it yet.
+ETC_20151224_017699	There are quite a few reckless people who are going after the monsters that have big ores like the Kenol attached onto the bodies of the monsters. It would have been better if they had acted more cautiously when thinking about the disaster that was brought to the world.
+ETC_20151224_017700	He has four strong arms. If you don't be careful, you may receive major damage.
+ETC_20151224_017701	A demon that follows the Demon Lord Hauberk. Most of them still show their loyalty to Hauberk even after their owner was sealed by Helgasercle.
+ETC_20151224_017702	Hohen competed against the humans since the birth of the kingdom. They were cooperative to demons instead of the goddesses, but they posed no big problems since their amount was small compared to humans. However, the environment that Medzio Diena changed is working more advantegeous for Hohens instead of humans so it is not certain whether humans would have an edge over them in the future.
```

Comment:
Hohen competed against the humans since....

Hohen competed **against humans** since...